content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx

```diff
@@ -71,6 +73,45 @@ CREATE TABLE "CategoriesOnPosts" (
 CREATE UNIQUE INDEX "CategoriesOnPosts_category_post_unique" ON "CategoriesOnPosts"("categoryId" int4_ops,"postId" int4_ops);
```

Comment:
Interesting that this Unique index seems to be missing on the real, generated SQL (see below). Could point to a logic error here.

Reply:
@Xula @janpio 
Is this example bellow more accurately?

```sql
CREATE TABLE "Category" (
    id SERIAL PRIMARY KEY
    name TEXT NOT NULL
);

CREATE TABLE "Post" (
    id SERIAL PRIMARY KEY
    title TEXT NOT NULL
);

CREATE TABLE "CategoriesOnPosts" (
    "categoryId" INTEGER NOT NULL,
    "postId" INTEGER NOT NULL,
    "assignedBy" TEXT NOT NULL,    
    "assignedAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
    FOREIGN KEY ("categoryId") REFERENCES "Category"("id") ON DELETE RESTRICT ON UPDATE CASCADE,
    FOREIGN KEY ("postId") REFERENCES "Post"("id") ON DELETE RESTRICT ON UPDATE CASCADE,
    CONSTRAINT "CategoriesOnPosts_pkey" PRIMARY KEY ("postId", "categoryId")
);
```

Reply:
@joaobispo2077 I think yes!